.github/workflows/docs-book.yml

```diff
@@ -15,10 +15,12 @@ jobs:
         uses: peaceiris/actions-mdbook@v1
         with:
           mdbook-version: '0.4.3'
+
       - run: mdbook build
+      - run: mdbook test
```

Comment:
does this actually works as expected? Running it locally it fails with a spectacular number of failures :o

Reply:
you got me :>. I'm trying to run it locally though, and failing spectacularly..

Reply:
also, for this to work we need to have [mdbook-linkcheck](https://github.com/Michael-F-Bryan/mdbook-linkcheck) and say so in the book.toml.